_docs_integrate/data-model-overview.md

```diff
@@ -353,6 +353,7 @@ In the [Attribute introduction]({% link _docs_integrate/attribute-introduction.m
 | succeededBy  | `string` \| `undefined`                                                                        | The `id` of a possibly existing successor of a LocalAttribute. It is possible to update the [Attribute](#attributes) contained in the `content` property of a LocalAttribute using [Attribute succession]({% link _docs_integrate/update-attributes-by-succession.md %}). In this case, a new LocalAttribute, the so-called successor, is created. Its `id` is then the value of the `succeededBy` property of the old LocalAttribute, the so-called predecessor. If a LocalAttribute has no successor, its `succeededBy` property is `undefined`. |
 | shareInfo    | [`LocalAttributeShareInfo`](#localattributeshareinfo) \| `undefined`                           | Information about the peer this LocalAttribute was received from/shared with, as well as via which LocalRequest it was received/sent. If the LocalAttribute refers to a RepositoryAttribute, this property is `undefined`.                                                                                                                                                                                                                                                                                                                         |
 | deletionInfo | [`LocalAttributeDeletionInfo`](#localattributedeletioninfo) \| `undefined`                     | Information about whether the counterpart of a shared Attribute is to be deleted or was deleted by the peer of the Relationship, as well as the point in time of that deletion.                                                                                                                                                                                                                                                                                                                                                                    |
+| isDefault    | `true` \| `undefined`                                                                          | States whether the LocalAttribute is the default RepositoryAttribute for its Attribute value type. For every IdentityAttribute value type exactly one RepositoryAttribute has `isDefault` set, given that at least one RepositoryAttribute of that value type exists. This property is only used for the App, e.g. to mark the Attribute that should be used by default, if multiple Attributes of the same value type exist. Thus, for a Connector it will always be `undefined`.                                                                 |
```

Comment:
> This property is only used for the App, e.g. to mark the Attribute that should be used by default, if multiple Attributes of the same value type exist.

I would have liked to see a more concrete example here that is related to the [Read Attributes from peer](https://enmeshed.eu/integrate/read-attributes-from-peer#accept-a-readattributerequestitem) guide or [Propose Attributes to peer](https://enmeshed.eu/integrate/propose-attributes-to-peer#accept-a-proposeattributerequestitem) guide.